src/gui/widgets/status_label_helper.hpp

```diff
@@ -62,7 +62,7 @@ default_status_value_getter(T& w)
  */
 template<typename W>
 std::function<void()> bind_status_label(
-		widget& find_in,
+		widget* find_in,
```

Comment:
I oppose this whole commit. If a parameter isn't supposed to be nullable, it should be a reference instead of a pointer (barring some rare exceptions).